documentation/modules/ROOT/pages/connectors/sqlserver.adoc

```diff
@@ -18,82 +21,121 @@ Want to help us further hone and improve it? link:/docs/contribute/[Learn how].
 // ModuleID: how-debezium-sql-server-connectors-work
 // Title: How {prodname} SQL Server connectors work
 [[how-the-sqlserver-connector-works]]
 == How the SQL Server connector works
 
+To optimally configure and run a {prodname} SQL Server connector, it is helpful to understand how the connector performs snapshots, streams change events, determines Kafka topic names, and uses metadata. 
+ifdef::product[]
+For details about how the connector works, see the following sections: 
+
+* xref:how-debezium-sql-server-connectors-perform-database-snapshots[]
+* xref:how-the-debezium-sql-server-connector-reads-change-data-tables[]
+* xref:default-names-of-kafka-topics-that-receive-debezium-sql-server-change-event-records[]
+* xref:how-the-debezium-SQL Server connector-uses-the-schema-change-topic[]
+* xref:descriptions-of-debezium-sql-server-connector-data-change-events[]
+* xref:debezium-sql-server-connector-generated-events-that-represent-transaction-boundaries[]
+* xref:how-debezium-SQL Server-connectors-map-data-types[]
+endif::product[]
+
 // Type: concept
 // ModuleID: how-debezium-sql-server-connectors-perform-database-snapshots
 // Title: How {prodname} SQL Server connectors perform database snapshots
 [[sqlserver-snapshots]]
 === Snapshots
 
-SQL Server CDC is not designed to store the complete history of database changes.
-It is thus necessary that {prodname} establishes the baseline of current database content and streams it to the Kafka.
-This is achieved via a process called snapshotting.
+SQL Server CDC is not designed to store a complete history of database changes.
+For the {prodname} SQL Server connector to establish a baseline for the current state of the database, 
+it uses a process called _snapshotting_.
 
-By default (snapshotting mode *initial*) the connector will upon the first startup perform an initial _consistent snapshot_ of the database
-(meaning the structure and data within any tables to be captured as per the connector's filter configuration).
+You can configure how the connector creates snapshots.
+By default, the connector's snapshot mode is set to `initial`. 
+Based on this `initial` snapshot mode, the first time that the connector starts, it performs an initial _consistent snapshot_ of the database.
+This initial snapshot captures the structure and data for any tables that match the criteria defined by the `include` and `exclude` properties that are configured for the connector (for example, `table.include.list`, `column.include.list`, `table.exclude.list`, and so forth).
 
-Each snapshot consists of the following steps:
+When the connector creates a snapshot, it completes the following tasks:
 
-1. Determine the tables to be captured
-2. Obtain a lock on each of the monitored tables to ensure that no structural changes can occur to any of the tables.
+1. Determines the tables to be captured.
+2. Obtains a lock on each of the monitored tables to ensure that no structural changes can occur to any of the tables.
```

Comment:
Here again the word `monitored` is used which was replaced in introduction. The terminologiy should be unified.

Reply:
We should generally use "capture" (discussed with @TovaCohen some time ago). "monitoring" refers to (JMX) monitoring and "tracking", yeah, it's potentially confusing.